types/rbac_test.go

```diff
@@ -1,6 +1,7 @@
 package types
 
 import (
+	"types"
```

Comment:
You can remove this line; we are already in the `types` package so no need to import it, otherwise it cause Go to fail!

Reply:
interesting! addressed, but the suite passed for me. what failure does this cause?

Reply:
@bitmonk See https://travis-ci.org/sensu/sensu-go/builds/401827783